CIP-XXXX/README.md

```diff
@@ -0,0 +1,141 @@
+---
+CIP: ?
+Title: Orthogonal Certificates
+Status: Proposed
+Category: Meta
+Authors:
+    - Joshua Marchand <josh@securitybot.info>
+Implementors: N/A
+Discussions:
+    - https://discord.gg/tsr7H6ApdT
+Created: 2025-04-13
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP proposes a simplified design of stake managmenet certificate types focused on simplicity and usability.
+
+The current Conway era ledger specification includes multiple certificate types each representing different combinations of stake registration, delegation, and vote delegation. This proposal simplifies this specification by defining a set of orthogonal certificate types that can be composed to achieve the same functionality, making the ledger more intuitive and easier to document, use, and maintain.
+
+## Motivation: why is this CIP necessary?
+
+The Conway era introduces multiple certificate types for various stake operations including combinations of:
+
+- Registering a stake credential
+- Delegating stake to a pool
+- Delegating voting power to a DRep
+
+This approach results in many complex and convoluted certificate types, increasing complexity for developers and users. The number of certificate types would also grow combinatorially with new features, making it only harder to document, extend, and understand the certificates.
+
+By redesigning the certificate types with orthogonal certificates, we can:
+ - Make the system more intuitive for users and developers
+ - Simplify documentation by having self-describing, single-purpose certificates
+ - Make ledger implementations easier to maintain
+ - Avoid combinatorial growth of certificates with future features
+
+
+## Specification
+
+### Current Certificates CDDL
+The Conway era has multiple certificate types for stake related operations:
+```
```

Comment:
```suggestion
```cddl
```

Reply:
committing to ensure proper syntax display for meeting tomorrow.